wandb/sdk/wandb_run.py

```diff
@@ -54,6 +54,7 @@
 )
 from wandb.util import (
     add_import_hook,
+    deprecate,
```

Comment:
can you put this in wandb.sdk.lib instead

Reply:
util isnt annotation checked at all

Reply:
sure, will move there. `util` is really big, type-annotating it seems worth a dedicated PR.